src/photosui.cs

```diff
@@ -24,6 +24,8 @@ namespace PhotosUI {
 	[iOS (8, 0)]
 	[Mac (10, 13)]
 	[Protocol]
+	[Unavailable (PlatformName.UIKitForMac)]
```

Comment:
@tj-devel709 Can you share more about `UIKitForMac` and `[Unavailable]`? :)

Reply:
Hi Whitney, do you mean in these github comments or in the PR?